actionmailbox/test/unit/mail_ext/addresses_test.rb

```diff
@@ -18,6 +19,10 @@ class AddressesTest < ActiveSupport::TestCase
       assert_equal "example.com", @mail.from_address.domain
     end
 
+    test "reply to address uses address object" do
+      assert_equal "example.com", @mail.reply_to_address.domain
```

Comment:
Why not just assert the exact email here?

```suggestion
      assert_equal "sarah@example.com", @mail.reply_to_address
```

Reply:
The point of these tests is to assert that `@mail.reply_to_address` is a `Mail::Address` (not a string!) and that it contains correctly parsed data.

That said, more ideal tests here might test other attributes of the `Mail::Address` object besides just `domain`, but I’m following along with how this is handled for the tests of `from_address`, `to_addresses`, etc. They all just look at the `domain` property as sort of a smoke test — I think the assumption here is that we should be able to trust that the Mail gem does the right thing, and these tests just need to make sure we are pulling the right value out of the right place. 🤷